### PR TITLE
included use cases to the download action documentation

### PR DESCRIPTION
**Pull request by ohansFavour**
Added use cases and caveats for the download action documentation.

### CODE REVIEW

File: framework-reference/download.md [hunk 1]
```diff
@@ -18,9 +18,32 @@ download(data: any, fileName: string, fileType?: string): void
...
 #### Arguments
 
-| **Argument Name** | **Description** |
-| :--- | :--- |
-| **data** | Data that would be downloaded |
-| **fileName** | Name of the file to be downloaded |
-| **fileType** | The mime type of the file to be downloaded |
+| **Argument Name** | **Description**                            |
+| :---------------- | :----------------------------------------- |
+| **data**          | Data that would be downloaded              |
```
Comment (hetunandu) [inline framework-reference/download.md:23]: data or url

**Review by hetunandu — CHANGES_REQUESTED** [review]
Small change

File: framework-reference/download.md (hunk 1 above)
Comment (ohansFavour) [reply framework-reference/download.md]: All right. Fixed in latest commit.

**Review by hetunandu — APPROVED** [review]

**Comment by vihar** [comment]
@ohansFavour, can you delete `.DS_Store` files on this commit?

**Comment by ohansFavour** [comment]
> @ohansFavour, can you delete `.DS_Store` files on this commit?

Sure

**Pull request merged**